en/04_Changelogs/5.0.0.md

```diff
@@ -447,58 +456,56 @@ Note that as part of this change the namespaces of several classes changed. Refe
 - The elemental [TopPage DataExtension](api:DNADesign\Elemental\TopPage\DataExtension) is applied to [`BaseElement`](api:DNADesign\Elemental\Models\BaseElement) and [`ElementalArea`](api:DNADesign\Elemental\Models\ElementalArea) by default, and the corresponding [`SiteTreeExtension`](api:DNADesign\Elemental\TopPage\SiteTreeExtension) is applied to [`SiteTree`](api:SilverStripe\CMS\Model\SiteTree) by default. This allows you to use `$TopPage` in your elemental block templates as a more performant alternative to `$Page` when referring to the page the element belongs to.
 - Void HTML elements such as `<img>` are no longer rendered as self-closing tags e.g. `<img src="foo" alt="bar">` is now rendered, previously `<img src="foo" alt="bar" />` was rendered.
-- It’s now possible to hook into FormField validation via the `updateValidationResult` extension hook. Module authors releasing form fields should opt-in to this functionality by calling the new `extendValidationResult` method to enable developers to extend and customise validation. See the [form validation documentation](/developer_guides/forms/validation/#extensions) for more information.
-
-## Bugfixes {#bugfixes}
-
-- If a page which is a child of a root-level page gets archived, and then its former parent is removed, it can only be restored if `can_be_root` for that page's class is true.
-
-This release includes a number of bug fixes to improve a broad range of areas. Check the change logs for full details of these fixes split by module. Thank you to the community members that helped contribute these fixes as part of the release!
+- It’s now possible to hook into `FormField` validation via the `updateValidationResult` extension hook. Module authors releasing form fields should opt-in to this functionality by calling the new `extendValidationResult` method to enable developers to extend and customise validation. See the [form validation documentation](/developer_guides/forms/validation/#extensions) for more information.
 
 ## Environment variable changes
 
-- `SS_MANIFESTCACHE` can no longer use the now removed symfony/cache 4.x "Simple" cache classes e.g. `Symfony\Component\Cache\Simple\PhpFilesCache`. Instead use the corresponding "Adapter" class e.g. `Symfony\Component\Cache\Adapter\PhpFilesAdapter`.
+- `SS_MANIFESTCACHE` can no longer use the now removed `symfony/cache` 4.x "Simple" cache classes e.g. `Symfony\Component\Cache\Simple\PhpFilesCache`. Instead, use the corresponding "Adapter" class e.g. `Symfony\Component\Cache\Adapter\PhpFilesAdapter`.
 - `APP_SMTP_USERNAME` and `APP_SMTP_PASSWORD` have been removed. Use a `MAILER_DSN` environment variable instead to configure SMTP email (see [the email documentation](/developer_guides/email/) for more details).
 
+## Bugfixes {#bugfixes}
```

Comment:
```suggestion
## Bug fixes {#bug fixes}
```

The automated changelog adds a `#bugfixes` anchor, so we need this one to be unique.
Also update the TOC.

Reply:
Done